src/main/java/uk/co/ramp/covid/simulation/population/Adult.java

```diff
@@ -59,6 +63,9 @@ public void allocateCommunalPlace(Places p) {
             } break;
             case HOSPITAL: {
                 primaryPlace = p.getNextHospitalJob();
+                if (!(primaryPlace instanceof CovidHospital)) {
```

Comment:
This isn't a specific request for changes, and what you have done here is precicely what I asked for in the Issue. However, that was a half-fudge because wht has really happenned is that some staff (clerical staff etc) have been told to work from home whilst a lot of clinical staff are on reduced shift patterns (week-on week-off etc). so what we might do in future is to change this so that the furloughed staff change each week. But let's not worry about that for now